### DISCUSSION TITLE
Temporary Shortcuts

### DISCUSSION THREAD

**Discussion by ChrisSimeon** [espanso/espanso#1179] [Feature requests and Ideas]
Hey! You know what would be great? Temporary Shortcuts. Let's say you right about a certain topic and you have to use this word 20 times while you're at it but afterwards never again. It would be great if you could mark that word and add it as a temporary shortcut. After you're done you press a combination, which clears all these shortcuts. 

Unfortunately I'm not familiar with Rust, but when I have the time maybe I can find a way to do this using the Terminal.

**Comment by federico-terzi** [comment]
Hey @ChrisSimeon 

Thank you for the feedback! I've been thinking about this exact idea for a while now and I definitely think it could be useful. I'll add it to the roadmap!

Cheers :)
